packages/storybook/src/migrations/update-13-4-5/set-project-build-config.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  logger,
+  Tree,
+  formatFiles,
+  updateProjectConfiguration,
+  getProjects,
+} from '@nrwl/devkit';
+
+export default async function setProjectBuildConfig(tree: Tree) {
+  let changesMade = false;
+  const projects = getProjects(tree);
+  [...projects.entries()].forEach(([projectName, projectConfiguration]) => {
+    if (
+      projectConfiguration?.targets?.storybook?.options?.uiFramework ===
+        '@storybook/angular' &&
+      projectName
+    ) {
+      if (!projectConfiguration.targets.storybook.options.projectBuildConfig) {
```

Comment:
What if my target is named `my-storybook` instead of the classic `storybook`? for instance if someone has something like this

```json
"my-project": {
        "my-storybook": {
          "builder": "@nrwl/storybook:storybook",
          "options": {
            "uiFramework": "@storybook/angular",
            "port": 4400,
            "config": {
              "configFolder": "libs/images/.storybook"
            }
          },
          "configurations": {
            "ci": {
              "quiet": true
            }
          }
        },
        "my-build-storybook": {
          "builder": "@nrwl/storybook:build",
          "outputs": ["{options.outputPath}"],
          "options": {
            "uiFramework": "@storybook/angular",
            "outputPath": "dist/storybook/images",
            "config": {
              "configFolder": "libs/images/.storybook"
            }
          },
          "configurations": {
            "ci": {
              "quiet": true
            }
          }
        }
}
```

This would not be captured by the migration. Could we do something like

- `target === '@nrwl/storybook:storybook'` => Storybook serve config
- `target === '@nrwl/storybook:build'` => Storybook build config